transmogrifier/config.py

```diff
@@ -83,10 +85,30 @@ def get_transformer(source: str) -> type:
     return getattr(source_module, class_name)
 
 
-def load_external_config(path: str) -> dict:
+def load_external_config(file_path: str, file_type: str) -> Union[dict, Tag]:
     """
-    Return dict from JSON config file.
+    Return dict from a config file.
 
     """
-    with open(path, "rb") as json_file:
-        return json.load(json_file)
+    if file_type == "json":
+        with open(file_path, "rb") as json_file:
+            return json.load(json_file)
+    elif file_type == "xml":
+        with open(file_path, "rb") as xml_file:
+            return BeautifulSoup(xml_file, "xml")
+    else:
+        return {}
+
+
+def create_dict_from_xml_config(
```

Comment:
I would probably specify in the function name that this is for MARC or LOC XML configs.